public/pages/CreateTrigger/containers/DefineTrigger/DefineTrigger.js

```diff
@@ -81,12 +81,14 @@ class DefineTrigger extends Component {
   componentDidMount() {
     const {
       monitorValues: { searchType, uri },
+      onRun,
     } = this.props;
     switch (searchType) {
       case SEARCH_TYPE.CLUSTER_METRICS:
         if (canExecuteClusterMetricsMonitor(uri)) this.onRunExecute();
         break;
       default:
+        onRun();
```

Comment:
Looking at this method inside the `ConfigureTriggers`, it seems to be calling the same endpoint with the same monitor data it receives as prop which is passed to this component. Can you explain a little more, why we need this call?